src/components/CoreLabel/index.tsx

```diff
@@ -1,22 +1,65 @@
 import CoreLogo from 'components/CoreLogo';
-import {SFC} from 'types';
+import {AppDispatch, Core, SFC} from 'types';
 import * as S from './Styles';
+import Line from 'components/Line';
+import {useToggle} from 'hooks';
+import {useDispatch, useSelector} from 'react-redux';
+import {getSelf} from 'selectors/state';
+import {deleteCore} from 'dispatchers/cores';
+import DropdownMenu from 'components/DropdownMenu';
+import CoreModal from 'modals/CoreModal';
+import {mdiDeleteOutline, mdiDotsVertical, mdiSquareEditOutline} from '@mdi/js';
```

Comment:
Same suggestion as above. 
To sort the code lines.

@buckyroberts I'm thinking we should document this somewhere in the code with some suggestions on how to do it well.